debian/wireguard-dkms.postinst

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+set -e
+
+#DEBHELPER#
+
+# Message of running rmmod & modprobe need to run after the module has been build. 
+
+case "$1" in
+	install)
+		;;
+
+	configure)
+		# Check if the wireguard module is loaded else the install will fail.
+		if [ ! -f "/sys/module/wireguard/version" ]; then
+			echo "WARNING: kernel module is not loaded"
```

Comment:
You don't need this WARNING, since the kernel module is loaded ondemand.

However, you do need the exit 0 below, so keep that.